resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -1,14 +1,20 @@
+# Where the puppet-agent dependency places puppet, facter, etc...
+# Puppet server expects to load Puppet from this location
+os-settings: {
```

Comment:
Yes, SERVER-331 involves getting rid of this section entirely, and yes it's fine to deal with that in a follow-up PR (I think that makes more sense).